portfolio/src/main/webapp/comments-script.js

```diff
@@ -37,6 +37,9 @@ function addCommentToDom(comments) {
         row.appendChild(createDataElement(comments[i].name));
         row.appendChild(createDataElement(comments[i].message));
         tableElement.appendChild(row);
+        if (i == commentsPerPage && commentsPerPage >0){
```

Comment:
You provide the possible comment values. Is the >0 check necessary since you only allow positive numbers?

Reply:
Why not put this check in the loop condition? i.e. for(... i<commensPerPage...)